server/guides/moderation/moderation.md

```diff
@@ -0,0 +1,45 @@
+# Moderating a Minecraft server
+
+Just as server settings are subject to debate, so is moderating a server. Everyone does it in their own way. But how efficient and good your server settings are is measurable. How well you moderate is much less measurable. Therefore, please see this part of the guide as well-intentioned advice and not as hard rules.
+
+## Rules and sanctions
+
+Before you can decide what rules to implement, it&#39;s important to determine what kind of server you want to run. After all, a family-friendly server for children has very different rules than an anarchy server. Often you will be looking for a balance between player freedom and player safety. For most servers, a good basis to adhere to is: a player&#39;s freedom stops when other players are bothered by actions.
```

Comment:
> `&#39`

Still present. Have you lintd these changes?

Reply:
i did. im probably running it wrong though